README.md

```diff
@@ -2,4 +2,4 @@
 
 **Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
 
-You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it
```

Comment:
kindly consider personalising your readme title so as it will be related to your project this will avoid confusion to people visiting your project

Reply:
- [ ] Keep doing the great work , really appreciate your feedback 👍